modules/rest-util/src/blaze/middleware/fhir/output.clj

```diff
@@ -53,6 +60,10 @@
   (cond-> response body (-> (update :body generate-xml)
                             (ring/content-type content-type))))
 
+(defn- encode-response-binary [{:keys [body] :as response} content-type]
```

Comment:
we have to take the content-type from the Binary resource

Reply:
https://www.hl7.org/fhir/R4/binary.html